### DISCUSSION TITLE
Which devices are supported?

### DISCUSSION THREAD

**Discussion by wojciech-kulik** [doronz88/pymobiledevice3#897] [Q&A] [answered]
I couldn't find this information. iOS, tvOS, watchOS, visionOS?

**Comment by doronz88** [comment, marked as answer]
Sorry I missed this post earlier. Technically that are all supposed to be supported, although I only have iPhones to test stuff on so only iOS is the officially tested.

**Reply by wojciech-kulik** [reply]
I tested it additionally on tvOS and it works :)
